service/access.go

```diff
@@ -233,6 +238,7 @@ func (a *accessService) StartAccessUpdater(ctx context.Context) <-chan error {
 	a.tokenCache.StartExpired(ctx, cachePurgePeriod)
 	a.tokenCache.EnableExpiredHook().SetExpiredHook(func(ctx context.Context, k string) {
 		glg.Warnf("the following cache is expired, key: %v", k)
+		glg.Warnf("Expired caches are included in the estimated memory usage being logged as INFO.")
```

Comment:
```suggestion
		glg.Warnf("the expired token data is still counted in the cache memory usage estimation even the allocated memory is freed, which causes over-estimation in the cache memory usage log message")
```

Reply:
also update previous log message
`glg.Warnf("the following cache is expired, key: %v", k)`
=>
`glg.Warnf("unexpected cache expiry, please review your refreshPeriod and expiry configuration, and related token expiry in the request body, key: %v", k)`

Reply:
@WindzCUHK fixed in https://github.com/AthenZ/athenz-client-sidecar/pull/18/commits/4d64861f9e57b8cd4c3c4bd6b03e6d65fb48dc8a